### PR TITLE
fix: update backup copy

### PR DESCRIPTION
**Pull request by gantunesr**
## **Description**

- Update the copy in the "Advanced Settings" to reflect the proper action.
- Update all the mentions of _backup_ (noun) to _back up_ (verb)

## **Related issues**

Fixes: https://github.com/MetaMask/accounts-planning/issues/141

## **Manual testing steps**

1. Go to Settings > Advanced
2. Check that the copy for the backup options are updated

### **Before**

Screenshot 2023-11-15 at 10 17 59

### **After**

Screenshot 2023-11-15 at 10 17 59

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - In case it's not yet "ready for review", I've set it to "draft".
  - In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -508,25 +508,25 @@
...
-    "message": "Backup now"
+    "message": "Back up now"
   },
   "backupUserData": {
-    "message": "Backup your data"
+    "message": "Back up your data"
   },
   "backupUserDataDescription": {
-    "message": "You can backup user settings containing preferences and account addresses into a JSON file."
+    "message": "You can back up data like your contacts and preferences into a JSON file."
```
Comment (danroc) [inline app/_locales/en/messages.json:529]: My 2¢:

> ... into a backup file.

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -3800,7 +3800,7 @@
     "message": "Restore user data"
   },
   "restoreUserDataDescription": {
-    "message": "You can restore user settings containing preferences and account addresses from a previously backed up JSON file."
+    "message": "You can restore data like contacts and preferences from a backed up JSON file."
```
Comment (danroc) [inline app/_locales/en/messages.json:3803]: > ... from a backup file.

File: app/_locales/en/messages.json (hunk 1 above)
Comment (gantunesr) [reply app/_locales/en/messages.json:529]: @coreyjanssen wdyt?

File: app/_locales/en/messages.json (hunk 2 above)
Comment (gantunesr) [reply app/_locales/en/messages.json:3803]: @coreyjanssen wdyt?

Comment (coreyjanssen) [reply app/_locales/en/messages.json:3803]: Also agree! `You can restore data like contacts and preferences from a backup file. `

File: app/_locales/en/messages.json (hunk 1 above)
Comment (gantunesr) [reply app/_locales/en/messages.json]: Solved in [f2cb351](https://github.com/MetaMask/metamask-extension/pull/21715/commits/f2cb3516c8af059835c4b8adfe8d24b20c433950)

File: app/_locales/en/messages.json (hunk 2 above)
Comment (gantunesr) [reply app/_locales/en/messages.json]: Solved in [f2cb351](https://github.com/MetaMask/metamask-extension/pull/21715/commits/f2cb3516c8af059835c4b8adfe8d24b20c433950)

**Review by danroc — DISMISSED** [review]

**Review by k-g-j — APPROVED** [review]
LGTM!
Screenshot 2023-11-15 at 9 43 13 AM

**Review by danroc — APPROVED** [review]

**Pull request merged**